bears/general/CLOCBear.py

```diff
@@ -0,0 +1,65 @@
+import re
+
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from dependency_management.requirements.DistributionRequirement import (
+    DistributionRequirement)
+
+
+@linter(executable='cloc')
+class CLOCBear:
+    """
+    Summarises a file or number of files in directory structure
+    with total lines, number of comment lines, number of actual code lines
+    using CLI tool cloc
+    """
+    LANGUAGES = {'All'}
+    REQUIREMENTS = {DistributionRequirement('cloc')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Number of comment lines',
```

Comment:
Bear.py file is part coala/coala project and not this. I'll just create a new issue referencing this one and create a PR for that first.
And will update the patch with proper aspect, of course.